gui.py

```diff
@@ -100,7 +100,11 @@ def setup(self, pfad_fenster_layout: str):
         ### GUI ###
         uic.loadUi(pfad_fenster_layout, self)
         self.setWindowIcon(QIcon(os.path.join(PATH, "images/spritze.ico")))
-        self.setWindowTitle('vaccipy ' + get_current_version())
+        try:
+            self.setWindowTitle('vaccipy ' + get_current_version())
+        except Exception as error:
+            #Standardwert ist vaccipy
```

Comment:
Muss der Standardwert gesetzt werden oder ist das by default so?

Reply:
Das ist bereits so in der .ui - Datei hinterlegt. Ich würde es jetzt aber trotzdem hin schreiben, um es klarer ersichtlich zu machen

Reply:
```suggestion
            self.setWindowTitle('vaccipy')
```